cron/auth02mozdef.py

```diff
@@ -224,37 +245,50 @@ def process_msg(mozmsg, msg):
             # Update a rule, add a site, update a site, etc
             details["description"] = msg.description
     except KeyError:
-        details["description"] = ""
+        pass
 
     # set the summary
-    if "auth" in mozmsg._category:
-        # make summary be action/username (success login user@place.com)
-        # include UNKNOWN as username value in summary
-        # if no details.username field exists
-        tmp_username = "UNKNOWN"
-        if 'username' in details:
-            tmp_username = details.username
-        mozmsg.summary = "{event} {username}".format(event=details.eventname, username=tmp_username)
-    else:
-        # default summary as action and description (if it exists)
-        mozmsg.summary = "{event} {desc}".format(event=details.eventname, desc=details.description)
+    # make summary be action/username (success login user@place.com)
+    # if no details.username field exists we don't add it.
+
+    # Build summary if neither email, description, nor username exists
+    if 'eventname' in details:
+        mozmsg.summary = "{event}".format(event=details.eventname)
+        if 'description' in details and details['description'] != "None":
```

Comment:
Syntactically, are we looking for the string "None" here or are we looking for an empty value? I don't think we often populate a value with "None"

Reply:
We are looking for the string "None" as that is what Auth0 is sending us. A field with a value of the string "None"
I mentioned this in the slack channel, that this was one of the things that was tripping up summary building until I modified how we do it.

Reply:
👍